BULK_UPLOAD_REVIEW.md

```
@@ -0,0 +1,116 @@
+# Bulk Upload Feature Review & Implementation
+
+## Backend Status
+
+### Persons/Employees
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)